internal/webservice/response.go

```diff
@@ -49,3 +49,19 @@ func errorResponse(w http.ResponseWriter, r *http.Request, err error) {
 		log.Logger.Errorf("could not write error response: %v", err)
 	}
 }
+
+func badRequestResponse(w http.ResponseWriter, r *http.Request, err error) {
+	log.Logger.Errorf("error processing request for %s: %v", r.URL.Path, err)
+	problemDetails := ProblemDetails{
+		Type:     "about:blank",
```

Comment:
I always thought it was silly that the RFC specifies this field, instead of just omitting it - this Is JSON, for god's sake!